client/src/pages/Login/Login.js

```diff
@@ -1,20 +1,60 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import styled from 'styled-components';
+import { useLocation, useNavigate } from 'react-router-dom';
+import searchImage from '../../assets/images/searchComponent.jpg';
+import { GOOGLE_LOGIN_REQUEST_URL, REDIRECT_URI } from '../../config';
+import { setLoginStatus, resetUserState } from '../../redux/users/reducer';
+import { googleLogoutAsync } from '../../redux/users/thunks';
+import { resetListState } from '../../redux/home/reducer';
+import { resetLikesState } from '../../redux/likes/reducer';
 
 function Login() {
-  const handleGoogleLogin = () => {
-    window.location.href = 'http://localhost:10010/users/google';
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const searchQuery = new URLSearchParams(useLocation().search);
+  const token = searchQuery.get('token');
+  const handleGoogleLogin = async () => {
+    dispatch(resetUserState());
+    dispatch(resetListState());
+    dispatch(resetLikesState());
+    try {
+      await dispatch(googleLogoutAsync());
```

Comment:
```suggestion
      await dispatch(googleLoginAsync());
```
I think it's a mistake.

Reply:
@pukkuKim It's not a mistake. before login, we force the prior user to log out to clear the session

Reply:
Ah, I see what's been causing the issue.
May I get the value for `GOOGLE_LOGIN_REQUEST_URL`?
I was stuck in a white blank page after attempting to login, so I thought it was because of the wrong function being called, but it was just not being redirected because I didn't have the variable in my .env.

Reply:
@pukkuKim  uploaded the .env on the notion page:
https://www.notion.so/thekimsplustwo/env-1338bb37ffa145359c6ed75a11cd87eb?pvs=4